packages/Webkul/Shop/src/Routes/customer-routes.php

```diff
@@ -73,24 +73,17 @@
                 ])->name('shop.customer.session.destroy');
 
                 /**
-                 * Wishlist.
-                 */
-                Route::post('wishlist/add/{id}', [WishlistController::class, 'add'])->name('shop.customer.wishlist.add');
-
-                Route::post('wishlist/share', [WishlistController::class, 'share'])->name('shop.customer.wishlist.share');
+                * Wishlist.
+                */
+                Route::get('wishlist', [WishlistController::class, 'index'])->name('shop.customers.account.wishlist.index');
 
-                Route::get('wishlist/shared', [WishlistController::class, 'shared'])
-                    ->defaults('_config', [
-                        'view' => 'shop::customers.account.wishlist.wishlist-shared',
-                    ])
-                    ->withoutMiddleware('customer')
-                    ->name('shop.customer.wishlist.shared');
+                Route::post('wishlist', [WishlistController::class, 'store'])->name('shop.customers.account.wishlist.add');
 
-                Route::delete('wishlist/remove/{id}', [WishlistController::class, 'remove'])->name('shop.customer.wishlist.remove');
+                Route::get('wishlist/move-to-cart/{id}', [WishlistController::class, 'moveToCart'])->name('shop.customers.account.wishlist.move_to_cart');
```

Comment:
This should be a post request.

`POST wishlist/{id}/move-to-cart`

Reply:
Done